packages/types/src/story/story.ts

```diff
@@ -20,7 +20,7 @@
 import type { Page } from './page';
 
 export interface Story {
-  version: number;
+  version?: number;
   pages: Page[];
   backgroundAudio?: {
     resource: {
```

Comment:
Somewhat unrelated, but while we are at it, this should be [audioResource](https://github.com/GoogleForCreators/web-stories-wp/blob/2fe143ba70ffb4720389b2bcfa7bcf3ae074de44/packages/types/src/resource/audioResource.ts).